Tools/autotest/arducopter.py

```diff
@@ -8483,6 +8483,21 @@ def AUTO_LAND_TO_BRAKE(self):
         self.change_mode('AUTO')
         self.wait_disarmed()
 
+    def MAV_LANDED_STATE_TAKEOFF(self):
```

Comment:
not in CAPS ! CAPITAL name are for constant ! (well there are some other test names like this but that isn't a good thing)